config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -354,6 +356,103 @@ presubmits:
       testgrid-tab-name: pr-cloud-provider-azure-e2e-ccm-capz
       description: "Runs Azure specific tests with cloud-provider-azure (https://github.com/kubernetes-sigs/cloud-provider-azure) using cluster-api-provider-azure."
       testgrid-num-columns-recent: '30'
+  - name: pull-cloud-provider-azure-e2e-ccm-vmss-capz
+    always_run: false
+    #skip_if_only_changed: "^docs/|^site/|^\\.github/|\\.(md|adoc)$|^(README|LICENSE)$"
```

Comment:
nit: remove comments

Reply:
When `always_run` is true, this line will be uncommented.
https://github.com/kubernetes/test-infra/pull/25202#discussion_r803348017